css/style.css

```diff
@@ -0,0 +1,73 @@
+body{
+	padding: 40px 20px;
+		font-family: sans-serif;
+	font-size: 18px;
+	border-top: 8px solid #79A6EA;
+	margin-top: 10px;
+	    color: #6C6C6C;
+} 
+
+
+body{
+	background: #79A6EA; /* For browsers that do not support gradients */
+    background: -webkit-linear-gradient(white, #79A6EA); /* For Safari 5.1 to 6.0 */
+    background: -o-linear-gradient(white, #79A6EA); /* For Opera 11.1 to 12.0 */
+    background: -moz-linear-gradient(white, #79A6EA;) /* For Firefox 3.6 to 15 */
+    background: linear-gradient(white, #79A6EA; )/* Standard syntax */
+}
+
+hr.style1 {
+	border-top: 4px double #79A6EA;
+	text-align: center;
+}
+
+
+
+p.inline {
+    display: inline;
+}
+p.rectangle {
+	border: 1px solid black;
+	padding: 15px;
+	margin-top: 10px;
+    display: inline-block;
+}
+
+
+h1 {
+	color:dark-grey;
+	font-size: 70px; 
+	text-align: center;}
+
+nav {text-align: center;
```

Comment:
:hammer: The code get's a bit difficult to read here (and below) because of indentation.